postgres/metadata.csv

```diff
@@ -27,6 +27,7 @@ postgresql.locks,gauge,,lock,,The number of locks active for this database.,0,po
 postgresql.seq_scans,gauge,,,,The number of sequential scans initiated on this table.,0,postgres,seq scans
 postgresql.seq_rows_read,gauge,,row,second,The number of live rows fetched by sequential scans.,0,postgres,seq rows rd
 postgresql.index_scans,gauge,,,,The number of index scans initiated on this table.,0,postgres,idx scans
+postgresql.index_rel_scans,gauge,,,,The number of index scans initiated on this table tagged by relation name.,0,postgres,idx scans
```

Comment:
Aren't you missing the type here ? https://help.datadoghq.com/hc/en-us/articles/206780123-Which-units-can-I-use-for-my-metrics-in-the-metric-summary-page-

Reply:
it's just a counter, we usually omit the unit in these cases